apps/backoffice-v2/src/domains/business-reports/query-keys.ts

```diff
@@ -11,6 +14,10 @@ export const businessReportsQueryKey = createQueryKeys('business-reports', {
     reportType: 'MERCHANT_REPORT_T1' & (string & {});
   }) => ({
     queryKey: [{ businessId, reportType }],
-    queryFn: () => fetchBusinessReports({ businessId, reportType }),
+    queryFn: () => fetchLatestBusinessReport({ businessId, reportType }),
+  }),
+  ongoingReportsList: ({ businessId }: { businessId: string }) => ({
+    queryKey: [{ businessId, reportType: 'ONGOING_MERCHANT_REPORT_T1' }],
+    queryFn: () => fetchBusinessReports({ businessId, reportType: 'ONGOING_MERCHANT_REPORT_T1' }),
```

Comment:
Why not pass `reportType`? As far as I know if its just a hardcoded string it doesn't need to be in `queryKey`.